internal/handlers/pg/pgdb/query_documents.go

```diff
@@ -97,6 +100,10 @@ func (pgPool *Pool) QueryDocuments(ctx context.Context, querier pgxtype.Querier,
 	}
 	sql += `FROM ` + pgx.Identifier{db, table}.Sanitize()
 
+	if sp.Explain {
```

Comment:
As `explain` doesn't return a huge list of docs (but rather an execution plan), does it make sense to include explain to this function (and deal with channels etc)? Or is it better to have a separate function for it?

Reply:
> As `explain` doesn't return a huge list of docs (but rather an execution plan), does it make sense to include explain to this function (and deal with channels etc)? Or is it better to have a separate function for it?

Answered in https://github.com/FerretDB/FerretDB/pull/909#discussion_r928533165